src/components/reusable/progressBar/progressBar.scss

```diff
@@ -0,0 +1,213 @@
+@use '../../../common/scss/global.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
```

Comment:
I might suggest importing the form-input.scss common styles, which can be leveraged for label, error, & option text for example for consistency.